### PR TITLE
added "slope graph" panel.  plugin code in netsage git

### PR DESCRIPTION
**Pull request by KatrinaTurner**

### CODE REVIEW

**Comment by daldoyle** [comment]
Seems to work when I manually install things.

Some things I'd like to see done before merging...

1 - add variable support for number of lines drawn. Right now it appears to be hardcoded for 10

2 - add variable support for the headers. Right now it appears to be hardcoded for src/dest organization. 

3 - update the Vagrant information so that it has the new plugin code as part of the build process, help others get bootstrapped

File: dashboards/flow-data-for-circuits.json [hunk 1]
```diff
@@ -1919,14 +1984,14 @@
           ]
         },
         "datasource": "netsage",
-        "definition": "{\"find\":\"terms\", \"field\":\"meta.sensor_id.keyword\", \"query\":\"-meta.sensor_id:*cenic* AND -meta.sensor_id:*pacificwave*\"}",
+        "definition": "{\"find\":\"terms\", \"field\":\"meta.sensor_id.keyword\", \"query\":\"(meta.flow_type:sflow OR meta.flow_type:netflow OR meta.flow_type:tstat) AND -meta.sensor_id:cenic*\"}",
```
Comment (daldoyle) [inline dashboards/flow-data-for-circuits.json]: Looks like a revert of an earlier commit, maybe this didn't get pulled or merged correctly if conflicts?

File: dashboards/flow-data-for-circuits.json [hunk 2]
```diff
@@ -1919,14 +1984,14 @@
...
         "hide": 0,
         "includeAll": true,
         "label": null,
         "multi": true,
         "name": "Sensors",
         "options": [],
-        "query": "{\"find\":\"terms\", \"field\":\"meta.sensor_id.keyword\", \"query\":\"-meta.sensor_id:*cenic* AND -meta.sensor_id:*pacificwave*\"}",
+        "query": "{\"find\":\"terms\", \"field\":\"meta.sensor_id.keyword\", \"query\":\"(meta.flow_type:sflow OR meta.flow_type:netflow OR meta.flow_type:tstat) AND -meta.sensor_id:cenic*\"}",
```
Comment (daldoyle) [inline dashboards/flow-data-for-circuits.json]: Same as above.

File: dashboards/flow-data-for-circuits.json [hunk 3]
```diff
@@ -382,7 +382,7 @@
...
+      "content": "<center><h1><b>Flow Data for Circuits</b></h1></center>\n<center>This dashboard provides flow data for the top ten sources and destinations by volume and rate for circuits.</center>\n<center>Click on an organization name in one of the tables to see the flow data dashboard for that organization.</center>\n<center>All times are displayed in browser local time.</center>\n<!-- Global site tag (gtag.js) - Google Analytics -->\n<script async src=\"https://www.googletagmanager.com/gtag/js?id=UA-142763676-1\"></script>\n<script>\n  window.dataLayer = window.dataLayer || [];\n  function gtag(){dataLayer.push(arguments);}\n  gtag('js', new Date());\n\n  gtag('config', 'UA-142763676-1');\n</script>",
```
Comment (daldoyle) [inline dashboards/flow-data-for-circuits.json]: Looks like this reverted an earlier commit, please fix.

Comment (KatrinaTurner) [reply dashboards/flow-data-for-circuits.json]: Should be fixed now! :)

**Comment by daldoyle** [comment]
This does not appear to work on first load, I get a view that looks all smashed up. If I reload or click Edit or something it renders fine. I think this is a regularly occurring issue and needs to be fixed before considering it in a release.

image

**Pull request closed without merging**